presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcOutputTableHandle.java

```diff
@@ -110,6 +111,11 @@ public String getTemporaryTableName()
         return temporaryTableName;
     }
 
+    public SchemaTableName getSchemaTableName()
```

Comment:
Inline. or make helper method in the cache class.

Once this method is added here, it will be much more expensive to remove it from here.

Reply:
@kokosing asked me to add this method here (was static before). I'm confused here.